spec/models/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+Monban.test_mode!
+
+describe User do
+  describe 'validations' do
+    it { should validate_presence_of(:email) }
+    it { should validate_presence_of(:password_digest) }
+  end
+
+  describe "when email format is invalid" do
+    it "should be invalid" do
+      user = create :user
+      addresses = %w[user@foo,com user_at_foo.org example.user@foo.]
+      addresses.each do |invalid_address|
+        user.email = invalid_address
+        expect(user).to_not be_valid
+      end
+    end
+  end
+
+  describe "when email format is valid" do
+    it "should be valid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.